WTWSMS/history/titles/e_byzantium.txt

```diff
@@ -1,15 +1,18 @@
-3.1.27={
+3.1.27= {
 	holder=145290
 	government = bureaucratic_government
 	law = succ_primogeniture
 	law = aristocracy_plutocracy_0
 	law = stability_2
 	law = agnatic_succession
 	law = centralization_3
-} # this way Jan 27 BC is displayed in his date, only backwards!
+	law = imperial_administration
+}
+# this way Jan 27 BC is displayed in his date, only backwards!
 24.8.19={
 	holder=145075
-} # moved date 10 years later so Augustus is not portrayed as child
+} 
+# moved date 10 years later so Augustus is not portrayed as child
```

Comment:
I didn't change anything. All I did was press the Enter key and move the  comment down by one.